packages/task/src/browser/task-service.ts

```diff
@@ -418,15 +418,15 @@ export class TaskService implements TaskConfigurationClient {
         }
     }
 
-    async runTaskByLabel(taskLabel: string): Promise<boolean> {
+    async runTaskByLabel(taskLabel: string): Promise<TaskInfo | undefined> {
```

Comment:
I am not sure if the code would always work properly in the multi root workspace, as tasks from different project folders could possibly have the same label. e.g., your "taskStart" and "taskEnd" might come from `tasks.json` in a different project folder.

Reply:
yes, I noticed the same and [commented](https://github.com/eclipse-theia/theia/pull/5913#pullrequestreview-277148313) it for the PR which brought this functionality